commons/zenoh-buffers/Cargo.toml

```diff
@@ -26,10 +26,11 @@ description = "Internal crate for zenoh."
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
 default = ["std"]
-shared-memory = []
+shared-memory = ["zenoh-shm"]
```

Comment:
I disagree making `zenoh-buffers` depending on such a complex crate like `zenoh-shm`.
`zenoh-buffers` should be and stay a core crate with no dependencies at all rather than the standard library and `zenoh-collections`.

Reply:
Fixed